tools/build_defs/scala/scala.bzl

```diff
@@ -95,6 +108,11 @@ def _collect_jars(ctx):
       jars += target.java.transitive_runtime_deps
   return jars
 
+def _args_for_suites(suites):
+  args = ["-o"]
+  for suite in suites:
+    args.extend(["-s", suite])
```

Comment:
are these needed? It would be nice to default to discovering all the suites in the jar and running them. That seems like the preferred way to go. If you don't want to run a suite, it should be in its own target, and just not test that target.

Or at least, that's my view.